mess110,2017-10-09T15:27:38Z,ACK https://github.com/bitcoin/bitcoin/pull/11468/commits/7481579131e3a330300d2c7f9a8f93aa67be33e1,https://github.com/bitcoin/bitcoin/pull/11468#issuecomment-335192448,335192448,
promag,2017-10-09T15:54:02Z,"utACK 7481579.\n\nI like seeing `Test X: PASS` and I in the same line `Test X: FAILED`. How about wrapping in try/catch instead?",https://github.com/bitcoin/bitcoin/pull/11468#issuecomment-335200191,335200191,
jnewbery,2017-10-09T17:49:31Z,"> I like seeing Test X: PASS and I in the same line Test X: FAILED. How about wrapping in try/catch instead?\n\nI don't think there's need for that. The problem this is solving is not having a way to tell where in the test script execution has reached when there's a failure. I've done that with the smallest change I could.\n\nWe should stop using the comparison test framework entirely for thes",https://github.com/bitcoin/bitcoin/pull/11468#issuecomment-335232696,335232696,
promag,2017-10-09T22:53:03Z,"Yes, that should be a bigger patch. Just read the IRC log üëç ",https://github.com/bitcoin/bitcoin/pull/11468#issuecomment-335311276,335311276,
MarcoFalke,2017-11-10T15:07:34Z,Obvious utACK 7481579. Sorry that I missed this.,https://github.com/bitcoin/bitcoin/pull/11468#issuecomment-343497559,343497559,
